### ISSUE TITLE
Reproducible bug: Cancelling reply causes undismissable notification 

### ISSUE THREAD

**Issue by ballo** [soduto/Soduto#31]
If I get a SMS notification (QKSMS in this case) and I hit reply, type something, then cancel the reply, the notification can not be dismissed and is displayed permanently.

**Comment by sannidhyaroy** [comment]
> If I get a SMS notification (QKSMS in this case) and I hit reply, type something, then cancel the reply, the notification can not be dismissed and is displayed permanently.

Just to confirm, you mean about the SMS notification from the `TelephonyService` that's also responsible for handling phone calls, and not the `NotificationService` that displays other phone notifications?

The reason I'm asking this is because the SMS notifications I get on my mac, is delivered by the `NotificationService`, just as other regular notifications. I thought the `TelephonyService` doesn't work with SMS anymore and hence bugs related to that went unnoticed.

Which phone are you using and which messages app does it use? (example: Google Messages or Samsung Messages)

**Comment by ballo** [comment]
They're just notifications created by soduto.

**Comment by ballo** [comment]
And as I said, it's QKSMS although it might be any notification that allows one to reply

**Comment by sannidhyaroy** [comment]
> They're just notifications created by soduto. 

Yes, but I'm asking which Service is responsible for creating the Notifications? And which phone is paired with Soduto?

**Comment by ballo** [comment]
> > They're just notifications created by soduto.
> 
> Yes, but I'm asking which Service is responsible for creating the Notifications? And which phone is paired with Soduto?

It's an S8. I don't know what you mean by "Service".

**Comment by ballo** [comment]
Here, try this: 

Steps to reproduce:
1) Receive SMS (or probably anything that you can reply to)
2) Hit the reply option in MacOS in the floating notification.
3) Type a bunch of garb then delete it (don't send the reply). 
4) You'll end up with a notification that you cannot dismiss (the 'X' goes away and never comes back)
